en/middleware/micrortps.md

```diff
@@ -1,24 +1,26 @@
 # RTPS/ROS2 Interface: PX4-FastRTPS Bridge
 
-The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *FastRTPS* applications.
+The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast-RTPS* applications, including ones built over the ROS2/ROS frameworks.
```

Comment:
eProsima call this *Fast RTPS* so we should probably use that rather than Fast-RTPS (or FastRTPS - which is what the original authors used).

Reply:
> applications, including ones built over the ROS2/ROS frameworks.

TO

> applications (including those built over the ROS2/ROS frameworks).

Reply:
Well they also have `Fast-RTPS` on their git repo name (https://github.com/eProsima/Fast-RTPS). But I am ok with using `FastRTPS`.